CorsixTH/Src/th_map_overlays.h

```diff
@@ -64,13 +64,13 @@ class map_typical_overlay : public map_overlay
     virtual ~map_typical_overlay();
 
     void set_sprites(sprite_sheet* pSheet, bool bTakeOwnership);
-    void set_font(THFont* font, bool take_ownership);
+    void set_font(::font* font, bool take_ownership);
```

Comment:
the need for `::` prefix signals to me that `font` may be too generic as class name.